modules/monitoring-moving-monitoring-components-to-different-nodes.adoc

```diff
@@ -175,5 +114,6 @@ Configurations applied to the `user-workload-monitoring-config` `ConfigMap` obje
 +
 [WARNING]
 ====
-When changes are saved to a monitoring config map, the pods and other resources in the related project might be redeployed. The running monitoring processes in that project might also be restarted.
+When you save changes to the `cluster-monitoring-config` config map, the pods and other resources in the openshift-monitoring project might be redeployed.
+The running monitoring processes in that project might also restart.
```

Comment:
since we have `cluster-monitoring-config`  and `user-workload-monitoring-config` configmap, the change is not suitable, no need to change, revert to the original description is fine